examples/autoload.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
I think this code sample needs an introduction of when to use this type of bootstrapping (in a file docblock).

Should probably also include a warning that using a mix of Requests 1.x and 2.x files is a bad idea to start with and to do so at your own risk.